database/tests/db_test.go

```diff
@@ -0,0 +1,143 @@
+package tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"reflect"
+	"testing"
+
+	"github.com/thrasher-/gocryptotrader/database"
+	"github.com/thrasher-/gocryptotrader/database/drivers"
+	dbpsql "github.com/thrasher-/gocryptotrader/database/drivers/postgres"
+	dbsqlite "github.com/thrasher-/gocryptotrader/database/drivers/sqlite"
+)
+
+var (
+	tempDir string
+	trueptr = func(b bool) *bool { return &b }(true)
+
+	postgresTestDatabase = database.Config{
+		Enabled:           trueptr,
+		Driver:            "postgres",
+		ConnectionDetails: drivers.ConnectionDetails{
+			//Host:     "",
+			//Port:     5432,
+			//Username: "",
+			//Password: "",
+			//Database: "",
+		},
+	}
+)
+
+func TestMain(m *testing.M) {
+	var err error
+	tempDir, err = ioutil.TempDir("", "gct-temp")
+
```

Comment:
Please remove spacing between err and err check

Reply:
done